docs/en/sql-reference/statements/delete.md

```diff
@@ -7,57 +7,81 @@ keywords: [delete]
 title: DELETE Statement
 ---
 
+The `DELETE` statement removes rows from the table `[db.]table` that match the expression `expr`.
+
 ``` sql
-DELETE FROM [db.]table [ON CLUSTER cluster] WHERE expr
+DELETE FROM [db.]table [ON CLUSTER cluster] WHERE expr;
 ```
 
-`DELETE FROM` removes rows from the table `[db.]table` that match the expression `expr`. The deleted rows are marked as deleted immediately and will be automatically filtered out of all subsequent queries. Cleanup of data happens asynchronously in the background. This feature is only available for the MergeTree table engine family.
+Deleted rows are internally marked as deleted immediately and will be automatically filtered out of all subsequent queries. Cleanup of data happens asynchronously in the background. The `DELETE` statement is only available for the *MergeTree table engine family.
```

Comment:
Should we specifically say that this is a "lazy" type of deletion and not that the deletion will happen asynchronously in the background, because we will not do anything to actively delete the data but will simply wait until the next merge happens? As a result, the deleted data can be stored for a long time without actually being deleted.